catalogs/hadoop-compatible-common/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+plugins {
+  id("java")
+}
+
+group = "org.example"
+version = "0.6.0-SNAPSHOT"
```

Comment:
Why are these two lines required and the license header is 2023?

Besides, I think it is not so elegant to have a new module for code sharing, is there a better solution to do it?

Reply:
Many JDBC-related catalogs share some common code and introduce a new catalog `catalog-jdbc-common`, I think we also do it. 

Besides, moving this module out of `catalogs` directory is not feasible. Currently, I haven't come up with a better one unless we will not share the code.

Reply:
Prefer not sharing code.

Reply:
@jerryshao  What's your thought about code sharing here, do we need to share kerberos-related code between different catalog modules? thanks.